packages/site/src/styles/global.js

```diff
@@ -74,4 +74,12 @@ export const GlobalStyle = createGlobalStyle`
   img {
     max-width: 100%;
   }
+
+  /* fix markdown link anchors due to fixed header */
```

Comment:
Let’s maybe add this to mdx.js instead? We’ll need to specifically add anchor links to headings anyway :)

Reply:
Good shout!